.github/workflows/build.yml

```diff
@@ -0,0 +1,97 @@
+on: push
+name: Build
+jobs:
+  build_macos:
+    name: Build (macOS)
+    runs-on: macos-latest
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v2
+    - name: Check out submodules
+      shell: bash
+      run: |
+        auth_header="$(git config --local --get http.https://github.com/.extraheader)"
+        git submodule sync --recursive
+        git -c "http.extraheader=$auth_header" -c protocol.version=2 submodule update --init --force --recursive --depth=1
+    - name: Install requirements
+      run: |
+        mkdir -p lib
+        cd lib
+        wget https://www.libsdl.org/release/SDL2-2.0.10.zip
+        wget https://www.libsdl.org/projects/SDL_mixer/release/SDL2_mixer-2.0.4.zip
+        unzip SDL2-2.0.10.zip
+        unzip SDL2_mixer-2.0.4.zip
+        cd SDL2-2.0.10
+        mkdir build
+        cd build
+        ../configure
+        make -j4
+        sudo make install
+        cd ../../
+        cd SDL2_mixer-2.0.4
+        mkdir build
+        cd build
+        ../configure
+        make
+        sudo make install
+    - name: Build
+      run: cd desktop_version/ && mkdir build && cd build && cmake .. && make
+    - name: Upload artifacts (MacOS)
+      uses: actions/upload-artifact@master
+      with:
+        name: mac
+        path: "desktop_version/build/vvvvvv.osx"
+  build_linux:
+    name: Build (Linux)
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v2
+    - name: Check out submodules
+      shell: bash
+      run: |
+        auth_header="$(git config --local --get http.https://github.com/.extraheader)"
+        git submodule sync --recursive
+        git -c "http.extraheader=$auth_header" -c protocol.version=2 submodule update --init --force --recursive --depth=1
```

Comment:
Why are you doing these steps? Seems unnecessary?

Reply:
Ah yes, not necessary any more. I was pulling in some CMake scripts for SDL from a submodule but removed that.